src/main/resources/resources/logisim/strings/hdl/hdl_de.properties

```diff
@@ -42,23 +42,23 @@ confirmCloseNo = Nein
 confirmCloseTitle = Bestätigen Sie den Abschluss
 confirmCloseYes = Ja
 hdlFrameTitle = Logisim-Evolution: HDL-Editor
-hexOpenErrorTitle = Öffnungsfehler
+hexOpenErrorTitle = Fehler beim Öffnen
 hexSaveErrorTitle = Fehler beim Speichern
 importMessage = Jede Änderung an dieser Komponente geht verloren. Trotzdem importieren?
 importOption = Importieren
 importTitle = Import bestätigen
-openButton = Importieren.....
```

Comment:
Can you please replace three dots with single ellipsis character -> `…`?

Reply:
I already talked with @maehne about this.
I decided to keep the three dots to be consistent between all (German) translations.

If we decide we want to use '…' then someone should do a PR, that changes it everywhere (when it makes sense).